bedrock/products/tests/test_views.py

```diff
@@ -506,7 +506,7 @@ def test_appropriate_active_locales_is_in_context(self, render_mock):
     def test_simple_get__no_active_locales_for_slug_at_all__gives_404(self):
         # change all entries so that get_active_locales helper will return []
         ContentfulEntry.objects.all().update(classification="something-that-will-not-match-query")
-        resp = self.client.get("/en-US/products/vpn/resource-center/slug-4/")
+        resp = self.client.get("/en-US/products/vpn/resource-center/slug-4/", follow=True)
```

Comment:
What's the reason behind why this needs to added `follow`?

Reply:
I think that's vestigial from the branch before the i18n refactor. Reverting now as the tests pass without it